ta/Makefile

```diff
@@ -5,6 +5,8 @@ ifeq ($O,)
 $(error output path should be specified when calling this makefile)
 endif
 
+LDFLAGS=
```

Comment:
This was particularly required to compile trusted application with yocto. For client application you can override LDFLAGS as well. From my understanding to compile trusted application we don't require external LDFLAGS as set by yocto environment. If I compile using make LDFLAGS=, client application fails to compile. Can you suggest proper way of doing this?